### PR TITLE
Update test addon creation to work better with new amo frontend.

### PR DESCRIPTION
**Pull request by jrbenny35**
* Create 2 new collections under the username 'Mozilla'. These have 1 addon each.
* Change the status of the installable addon to Public.

These changes help compatibility with the new amo front-end for integration testing.

Also I have created some variables that will help keep the tests updated as the new frontend changes.

### CODE REVIEW

File: src/olympia/landfill/serializers.py [hunk 1]
```diff
@@ -241,3 +284,6 @@ def create_installable_addon(self):
             # And let's create a new version for that upload.
             create_version_for_upload(
                 upload.addon, upload, amo.RELEASE_CHANNEL_LISTED)
+
+            # Change status to public
+            addon.update(status=amo.STATUS_PUBLIC)
```
Comment (EnTeQuAk) [inline src/olympia/landfill/serializers.py]: It wasn't already public? You shouldn't be able to see the add-on if it's not unless we're logging in with some user?

Comment (jrbenny35) [reply src/olympia/landfill/serializers.py]: No, you had to navigate to the URL to find the addon.

**Comment by jrbenny35** [comment]
@EnTeQuAk I can merge this myself?

**Comment by EnTeQuAk** [comment]
Absolutely if you have the permissions but I think you should. Just make
sure to squash those changes into one commit before merging

On Tue, Jan 23, 2018, at 8:09 PM, Benjamin Forehand Jr wrote:
> @EnTeQuAk[1] I can merge this myself?


> — You are receiving this because you were mentioned. Reply to this
> email directly, view it on GitHub[2], or mute the thread[3].> 

Links:

  1. https://github.com/entequak
  2. https://github.com/mozilla/addons-server/pull/7334#issuecomment-359896923

**Pull request merged**